.github/actions/deploy-frontend/action.yml

```diff
@@ -75,3 +75,17 @@ runs:
       run: |
         az storage blob delete-batch --account-name ${{ env.IMAGE_REPO }} -s '$web'
         az storage blob upload-batch --account-name ${{ env.IMAGE_REPO }} -s frontend-react/build -d '$web'
+
+    - name: Download Backend Artifact (prime-router)
+    - uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a
+      with:
+        name: prime-router-build-${{ inputs.version }}
+        path: prime-router
+
+    - name: Upload Swagger UI (APIDocs staging only)
+      if: inputs.env-name == 'staging'
+      shell: bash
+      working-directory: .
+      run: |
+        az storage blob delete-batch --account-name swaggeruiapidocs -s '$web'
```

Comment:
`swaggeruiapidocs` should be an env var, like in the command for react? Also, should be environment specific suffix, like `swaggeruiapidocs-stg`

Reply:
Was `swaggeruiapidocs` manually created? Need to open ticket with DevOps so it can be created with Terraform. Please link that ticket to the PR description when created.

Reply:
> Was `swaggeruiapidocs` manually created? Need to open ticket with DevOps so it can be created with Terraform. Please link that ticket to the PR description when created.

Yea, as Jeremy and you pointed out
Originally I prefer a initial manual provision, then github actions take care the on going update...

Now I know the convention, so will terraform it...

Reply:
FYI, DevOp will take care the terraforming part under another ticket